orchagent/aclorch.cpp

```diff
@@ -2706,23 +2717,22 @@ bool AclOrch::addAclTable(AclTable &newTable)
     // Check if a separate mirror table is needed or not based on the platform
     if (newTable.type == ACL_TABLE_MIRROR || newTable.type == ACL_TABLE_MIRRORV6)
     {
-
         if (m_isCombinedMirrorV6Table &&
```

Comment:
@daall should'nt m_isCombinedMirrorV6Table be also define per stage ? Possible Ingress can be separate and combine on egressor vice-versa

Reply:
I'm inclined to leave it alone for now since this variable is hardcoded based on platform: https://github.com/Azure/sonic-swss/blob/727a518159a08e245f6f37d58f796f2b9097774a/orchagent/aclorch.cpp#L2169

Reply:
ok